lib/src/bridge/bluetooth_state_mixin.dart

```diff
@@ -1,9 +1,11 @@
 part of _internal;
 
 mixin BluetoothStateMixin on FlutterBLE {
-  final Stream<dynamic> _adapterStateChanges =
-      const EventChannel(ChannelName.adapterStateChanges)
-          .receiveBroadcastStream();
+  final Stream<String> _adapterStateChanges =
+    const EventChannel(ChannelName.adapterStateChanges)
+      .receiveBroadcastStream()
+      .where((event) => event is String)
```

Comment:
There shouldn't be any filtering on this stream. If for any reason an unexpected value came, it should be raised as an error.